src/dlgProfilePreferences.cpp

```diff
@@ -268,7 +268,23 @@ dlgProfilePreferences::dlgProfilePreferences(QWidget* pF, Host* pHost)
     comboBox_guiLanguage->model()->sort(0);
     auto current = pMudlet->mInterfaceLanguage;
     comboBox_guiLanguage->setCurrentText(pMudlet->mLanguageCodeMap.value(current).first);
-    connect(comboBox_guiLanguage, QOverload<const QString &>::of(&QComboBox::currentIndexChanged), this, &dlgProfilePreferences::slot_changeGuiLanguage);
+    connect(comboBox_guiLanguage, QOverload<const QString&>::of(&QComboBox::currentIndexChanged), this, &dlgProfilePreferences::slot_changeGuiLanguage);
+
+
+    connect(registerTelnetURI, &QAbstractButton::clicked, this, &dlgProfilePreferences::slot_setURIMudlet);
+
+#if defined(Q_OS_WIN)
+    //registerTelnetURI->setEnabled(false);
+
+#elif defined(Q_OS_LINUX)
+    registerTelnetURI->setEnabled(true);
+#elif defined(Q_OS_FREEBSD)
+    registerTelnetURI->setEnabled(false);
```

Comment:
I will test this when I can get around to booting FreeBSD

Reply:
I don't think it does anything there right now... hold off for now.

Reply:
I mean it should be very similar - I hope...